net-libs/pacparser/pacparser-1.3.7-r1.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
what about 3.8?

Reply:
I could not test on the machine at hand, no reason to believe it would not work.

Reply:
I hope someone else with a test setup at hand, or the maintainer will add it eventually. Or i will do once 3.8 hits me.

Reply:
and because of that enews

```
2020-04-22-python3-7
  Title                     Python 3.7 to become the default target
  Author                    Michał Górny <mgorny@gentoo.org>
  Posted                    2020-04-22
  Revision                  1

```